#include "constants.hpp"
double NAV_RATE = 10;
double GUIDE_RATE = 1;

double SCSIM_ATT_ERROR = 0.2;
double SCSIM_POS_ERROR = 0.2;
double SCSIM_ATT_RATE = 1;
double SCSIM_POS_RATE = 1;

double TEST_ERROR = 0.05;
double TEST_RATE = 2;

double SMALL_ERROR = 0.3;
double SMALL_VELOCITY = 0.01;
double BIG_VELOCITY = 0.01;
double KP_DEFAULT = 1.5;
double KD_DEFAULT = 4;
double KP_BIG = 10;
double KD_BIG = 10;
